app/Http/Controllers/loginController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\forgetPasswordRequest;
+use App\Http\Requests\LoginRequest;
+use App\Http\Requests\setPasswordRequest;
+use App\Jobs\resetPasswordEmailJob;
+use App\Models\Resetpassword;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\Hash;
+
+class loginController extends Controller
+{
+    public function checkLogin(LoginRequest $req)
+    {
+        if (!$user = Auth::attempt(['email' => $req->email, 'password' => $req->password])) {
+            return errorResponse(trans('lang.Invalid_Credentials'), 401);
```

Comment:
Don't pass status code, U already included status code in helper success response